src/client/game.cpp

```diff
@@ -3592,7 +3585,7 @@ void Game::handleDigging(const PointedThing &pointed, const v3s16 &nodepos,
 		client->setCrack(-1, nodepos);
 	}
 
-	camera->setDigging(0);  // left click animation
+	camera->setDigging(0);  // Dig animation
```

Comment:
Another problem with the naming. Digging is also the name of the player model's hand movement.

Reply:
Agreed.

\*sigh\*, the naming issues are likely going to stall this PR big-time...

Reply:
Ah, are you referring to the function name?

Reply:
Thoughts on renaming this to `Camera::setAnim`? Maybe we can also pass an Enum as the parameter, instead of a plain number.